taxi/admin.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+@admin.register(Driver)
+class DriverAdmin(admin.ModelAdmin):
+    list_display = UserAdmin.list_display + ("license_number",)
+    fieldsets = UserAdmin.fieldsets + (("Additional info",
+                                        {"fields": ("license_number",)}),)
+    add_fieldsets = (UserAdmin.add_fieldsets +
+                     (("Additional info", {"fields": ("license_number",)}),))
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    search_fields = ["model", ]
```

Comment:
[Not a mistake] but this is not a tuple you don't really need to add a comma when you have only one element